.github/workflows/nodejs.yml

```
@@ -11,10 +11,10 @@ jobs:
         node: ['12', '13', '14', '15', '16', '17']
```

Let's drop version 13, since it's not an LTS version: https://nodejs.org/en/about/releases/
```suggestion
        node: ['12', '14', '15', '16', '17']
```

---

Sounds good, will commit changes shortly!